employees/tests/test_unit_export_data.py

```diff
@@ -97,3 +100,28 @@ def test_hours_should_be_the_same_in_excel(self):
 
     def test_description_should_be_the_same_in_excel(self):
         self.assertEqual(self.report.description, self.workbook_for_user.active.cell(row=2, column=5).value)
+
+
+class DataSetUpForSortingTests(TestCase):
```

Comment:
I think that as long as this is used only in single subclass, there is no need to keep it in separate class.